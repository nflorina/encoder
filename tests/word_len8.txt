abcdefgh
1
END
