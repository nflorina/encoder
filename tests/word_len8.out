1 0 0
efghabcd
efghabcd
